site/src/components/Footer.tsx

```diff
@@ -87,38 +83,67 @@ const SOCIALS: { name: string; icon: ReactNode; href: string }[] = [
     icon: <Icon className="fab fa-discord" />,
     href: "https://discord.gg/PefPteFe5j",
   },
+  {
+    name: "GitHub",
+    icon: <Icon className="fab fa-github" />,
+    href: "https://github.com/blockprotocol",
+  },
 ];
 
 const Socials = (
-  <Box mt={3} display="flex">
-    {SOCIALS.map(({ href, icon }) => (
-      <Link
-        href={href}
-        key={href}
-        sx={{
-          "&:not(:first-child)": {
-            marginLeft: 2.5,
-          },
-          ":hover": {
-            svg: {
-              color: (theme) => theme.palette.gray[30],
+  <Box
+    mt={3}
+    display="flex"
+    sx={{
+      flexDirection: {
+        xs: "column",
+        md: "row",
+      },
+      alignItems: { xs: "left", md: "center" },
+    }}
+  >
+    <Box marginBottom="1rem">
+      {SOCIALS.map(({ href, icon }) => (
+        <Link
+          href={href}
+          key={href}
+          sx={{
+            marginRight: 2.5,
+            ":hover": {
+              svg: {
+                color: (theme) => theme.palette.gray[30],
+              },
+            },
+            ":active": {
+              svg: {
+                color: (theme) => theme.palette.common.white,
+              },
             },
-          },
-          ":active": {
             svg: {
-              color: (theme) => theme.palette.common.white,
+              transition: (theme) =>
+                theme.transitions.create("color", { duration: 150 }),
+              color: (theme) => theme.palette.gray[50],
             },
-          },
-          svg: {
-            transition: (theme) =>
-              theme.transitions.create("color", { duration: 150 }),
-            color: (theme) => theme.palette.gray[50],
-          },
+          }}
+        >
+          {icon}
+        </Link>
+      ))}
+    </Box>
+    <a href="https://github.com/blockprotocol/blockprotocol">
```

Comment:
If I understand correctly, the new button pretty much replicates the GitHub social icon. It points to https://github.com/blockprotocol/blockprotocol instead of https://github.com/blockprotocol, but maybe the social link is wrong and should be pointing directly to the monorepo. There’s not much else in the org.

Not sure if it’s just me, but I’ve been always afraid of pressing custom-looking _fork_ and _star_ buttons. They feel like you’d make an action right away, without any further confirmation. Seeing the repo makes sense first, so I tend to not press them.

Perhaps, we could use widgets that look more GitHub-y? Here’s an example of what I mean: https://prettier.io

<img width="929" alt="Screenshot 2022-01-04 at 14 45 52" src="https://user-images.githubusercontent.com/608862/148076422-916b2106-fa71-4c58-9a16-06b776f027df.png">

Buttons like that still just take you to the repo because GitHub does not allow XSS, but they are subjectively less scary to press. You can also click the number of stargazers, which would take you to https://github.com/blockprotocol/blockprotocol/stargazers.

Here are some unofficial GitHub widgets, but I’m sure there are more:
- https://buttons.github.io
- https://ghbtns.com

All this is optional, especially before the launch. Just sharing some random thoughts 🙂

Reply:
Appreciate the thoughts! We could consider adding official github buttons later on - agree it might lead to a greater sense of trust / less mystery about what the button does.  For the moment we intentionally decided to leave the star count off until it's 100+